tracer/src/Datadog.Trace.Trimming/Datadog.Trace.Trimming.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Do we need this file if we have the `.nuspec`?

Reply:
`dotnet` CLI doesn’t support reading the nuspec file directly. So to avoid using the nuget binary I use this workaround

Reply:
Ah, thanks.